src/components/Scaler/index.js

```diff
@@ -81,30 +94,36 @@ class Scaler extends Component {
       const updateDomainsForItem = item => {
         domainsByItemId[item.id] = {
           [Axes.time]: this.props.dataContext.timeDomain || [
-            ...Axes.time(
-              this.state.domainsByItemId[item.id] || item.timeDomain
-            ),
+            ...(stripPlaceholderDomain(
```

Comment:
if this one returns `undefined` wouldn't spread operator fail?

Reply:
Yep. Though now that I look at it, `item.timeDomain` doesn't _necessarily_ exist (it's just that no one has ever used it .. yet). I guess I should add it to `DataProvider` too.